files/alert_rules.yml

```diff
@@ -258,15 +258,16 @@ groups:
             The alert will be resolved when '{{ $labels.module }}' is up and running
             and can be contacted by the admin machine.
 
-      - alert: Multiple coexisting software versions
-        expr: count(count(netdata_command__average{dimension=~'.*version'}) by (dimension, chart)) by (dimension) != 1
+      - alert: MultipleVersions
+        expr: count(count(label_replace(netdata_command__average{dimension=~'.*version'}, "software", "$1", "dimension", "([^_]+)_version")) by (software, chart)) by (software) > 1
```

Comment:
wrong regex to replace `xxx_version` with `xxx`.

Please use `(.+)` instead of `([^_]+)` because it excludes packages names with underscore inside (like `zk_lib`).

Reply:
good catch